presto-delta/src/main/java/com/facebook/presto/delta/DeltaSessionProperties.java

```diff
@@ -50,6 +53,15 @@ public DeltaSessionProperties(
                         "Enable cache for Delta tables",
                         cacheConfig.isCachingEnabled(),
                         false),
+                new PropertyMetadata<>(
+                        NODE_SELECTION_STRATEGY,
+                        "Node affinity selection strategy",
+                        VARCHAR,
+                        NodeSelectionStrategy.class,
+                        hiveClientConfig.getNodeSelectionStrategy(),
+                        false,
+                        value -> NodeSelectionStrategy.valueOf((String) value),
```

Comment:
`valueOf(((String) value).toUpperCase())`

Reply:
NodeSelectionStrategy is an Enum and doesn't have any explicit implementation for valueOf for importing

Reply:
what I meant was, converting the String `value` to uppercase so that even if it is `hard_affinity` instead of `HARD_AFFINITY` the call succeeds.

Reply:
Yes, make sense